core/src/actors/peers_manager.rs

```diff
@@ -20,6 +25,37 @@ pub struct PeersManager {
     peers: Peers,
 }
 
+impl PeersManager {
+    /// Method to periodically dump peers into storage
+    fn storage_peers(&self, ctx: &mut Context<Self>, storage_peers_period: Duration) {
+        // Schedule the discovery_peers with a given period
+        ctx.run_later(storage_peers_period, move |act, ctx| {
```

Comment:
If `act` stands for actor, I'd rather use the whole word.
I'm OK with `ctx` because that's a generally accepted and well-known apocope.

Reply:
I disagree. We are already using `|act, ctx|` in many places in the code so there is no point in changing it now. In the actix documentation they use `act` so I think it's better to use their conventions.

Reply:
![image](https://user-images.githubusercontent.com/1316439/48337947-adb15c00-e664-11e8-8de7-2f9d99ef6e1d.png)